Tensile/KernelWriterAssembly.py

```diff
@@ -10867,6 +10921,11 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
         vc1 = element[2]
         vc0 = element[3]
         sumIdx = ss.elementSumIdx[elementIdx]
+        if self.serializedStore:
+          # ensure index aligned by number of registers per element
+          assert(sumIdx%self.ss.cfg.numVgprPerValuC==0)
+          # convert from index in terms vgprs (serializedStore==True) to index in terms of elements (classic)
+          sumIdx = sumIdx//self.ss.cfg.numVgprPerValuC
```

Comment:
Looks the same as the above block, can they be combined?

Reply:
Thanks for the feedback. The unit conversion (vgpr index to element index) is now done somewhere else so they are all gone now.